### ISSUE TITLE
Zigpy Error in API.py in HA and no longer will integration work

### ISSUE THREAD

**Issue by Austin1** [zigpy/zigpy-deconz#246]
Not sure what happened but me and other users of home assistant have our zigbee networks down. I've been researching and researching and finally got an debug log that made some sense and that led me to this repo, which I see was just updated a week ago. Makes me wonder if the update broke the integration in HA. 

Here is the error I'm getting and the zigbee integration fails to load in HA:

```
2024-01-09 21:57:14.935 DEBUG (MainThread) [homeassistant.components.zha] Failed to set up ZHA
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/zigpy_deconz/api.py", line 586, in _command
    return await fut
           ^^^^^^^^^
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/zha/__init__.py", line 163, in async_setup_entry
    zha_gateway = await ZHAGateway.async_from_config(
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/zha/core/gateway.py", line 193, in async_from_config
    await instance.async_initialize()
  File "/usr/src/homeassistant/homeassistant/components/zha/core/gateway.py", line 211, in async_initialize
    await app.startup(auto_form=True)
  File "/usr/local/lib/python3.11/site-packages/zigpy/application.py", line 225, in startup
    await self.connect()
  File "/usr/local/lib/python3.11/site-packages/zigpy_deconz/zigbee/application.py", line 97, in connect
    await api.connect()
  File "/usr/local/lib/python3.11/site-packages/zigpy_deconz/api.py", line 466, in connect
    await self.version()
  File "/usr/local/lib/python3.11/site-packages/zigpy_deconz/api.py", line 802, in version
    self._protocol_version = await self.read_parameter(
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/zigpy_deconz/api.py", line 821, in read_parameter
    rsp = await self.send_command(
          ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/zigpy_deconz/api.py", line 508, in send_command
    return await self._command(cmd, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/zigpy_deconz/api.py", line 585, in _command
    async with asyncio_timeout(COMMAND_TIMEOUT):
  File "/usr/local/lib/python3.11/asyncio/timeouts.py", line 111, in __aexit__
    raise TimeoutError from exc_val
TimeoutError
```

**Comment by vrouleau** [comment]
I think I have the same problem. Got a ConBee III ( firmware 0x264f0900, HA 2024.1.5, docker test installation). This was during init of the Conbee key, after having selected to erase /create new network. Also got this stack:
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/zigpy/device.py", line 189, in initialize
    await self._initialize()
  File "/usr/local/lib/python3.11/site-packages/zigpy/util.py", line 131, in retry
    return await func()
           ^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/zigpy/device.py", line 210, in _initialize
    await self.get_node_descriptor()
  File "/usr/local/lib/python3.11/site-packages/zigpy/device.py", line 175, in get_node_descriptor
    status, _, node_desc = await self.zdo.Node_Desc_req(self.nwk)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/zigpy/device.py", line 323, in request
    await send_request()
  File "/usr/local/lib/python3.11/site-packages/zigpy/application.py", line 833, in request
    await self.send_packet(
  File "/usr/local/lib/python3.11/site-packages/zigpy_deconz/zigbee/application.py", line 508, in send_packet
    await self._api.aps_data_request(
  File "/usr/local/lib/python3.11/site-packages/zigpy_deconz/api.py", line 871, in aps_data_request
    rsp = await self.send_command(
          ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/zigpy_deconz/api.py", line 508, in send_command
    return await self._command(cmd, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/zigpy_deconz/api.py", line 577, in _command
    self._uart.send(command.replace(seq=seq).serialize())
    ^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'send'

**Comment by technicallyfeasible** [comment]
I am also getting errors with Conbee III on the latest 2 firmware versions (264e0900 and 0x264f0900):
```
2024-02-02 23:02:05.734 WARNING (MainThread) [zigpy_deconz.api] Failed to parse command Command(command_id=<CommandId.update_neighbor: 29>, seq=43, payload=b'\x00\x06\x00\x01')
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/zigpy_deconz/api.py", line 617, in data_received
    params, rest = t.deserialize_dict(command.payload, rx_schema)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/zigpy_deconz/types.py", line 50, in deserialize_dict
    result[name], data = type_.deserialize(data)
                         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/zigpy/types/basic.py", line 194, in deserialize
    raise ValueError(f"Data is too short to contain {byte_size} bytes")
ValueError: Data is too short to contain 2 bytes
```

**Comment by puddly** [comment]
This has been fixed by #247.
